tests/test_portchannel.py

```diff
@@ -57,3 +57,178 @@ def test_Portchannel(self, dvs, testlog):
 
         lagms = lagmtbl.getKeys()
         assert len(lagms) == 0
+
+
+    def test_Portchannel_oper_down(self, dvs, testlog):
+
+        self.adb = swsscommon.DBConnector(1, dvs.redis_sock, 0)
+        self.cdb = swsscommon.DBConnector(4, dvs.redis_sock, 0)
+        self.pdb = swsscommon.DBConnector(0, dvs.redis_sock, 0)
+
+        # Create 4 PortChannels
+        tbl = swsscommon.Table(self.cdb, "PORTCHANNEL")
+        fvs = swsscommon.FieldValuePairs([("admin_status", "up"),("mtu", "9100"),("oper_status", "up")])
+
+        tbl.set("PortChannel001", fvs)
```

Comment:
I see that there is code repetition. If you can create a list of Portchannels and call function to do set, del operation, it will simplify and reduce the code. Just a suggestion.